registry/registry.go

```diff
@@ -70,12 +70,16 @@ type NotFoundError struct {
 
 func (e *NotFoundError) Error() string {
 	var reqStr string
-	if len(e.Requests) == 1 {
-		reqStr = fmt.Sprintf("%q", e.Requests[0])
-	} else {
-		reqStr = strutil.Quoted(e.Requests)
+	switch len(e.Requests) {
+	case 0:
+		// leave empty, so the message reflects the request gets the whole view
```

Comment:
is there a test that hits this case in the package ?

Reply:
Only in the get_test.go. I've added one now